packages/vite/src/node/plugins/css.ts

```diff
@@ -462,60 +458,51 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
       if (!chunkCSS) {
         return null
       }
-
       const cssEntryFiles = cssEntryFilesCache.get(config)!
-      const publicAssetUrlMap = publicAssetUrlCache.get(config)!
 
       // resolve asset URL placeholders to their built file URLs
-      function resolveAssetUrlsInCss(chunkCSS: string, cssAssetName: string) {
-        const encodedPublicUrls = encodePublicUrlsInCSS(config)
-
+      const resolveAssetUrlsInCss = (
```

Comment:
I think we don't need to modify this function in this PR, but directly call `renderAssetUrl(` in line 546 with `hostType` `'js'`.